### PR TITLE
Added initial draft page for London 2016 event

### PR DESCRIPTION
**Pull request by barrychandler**
I've added an initial draft page for the London 2016 event

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
Hi, @barrychandler - there are merge conflicts with this PR. Also, you're removing items you shouldn't be (like proposals from other, past events). Please look over this PR and correct or replace it. Let us know if you'd like assistance. Thanks.

File: site/content/_future.txt [hunk 1]
```diff
@@ -1,3 +1,32 @@
+<<<<<<< HEAD
```
Comment (bridgetkromhout) [inline site/content/_future.txt]: Hi, @barrychandler -

Please take a look at this in your editor - you're introducing a lot of unneeded additions which will break this part of the site.

File: site/content/events/2011-bangalore/proposals/Agile catalyst : Devops, It starts at Developers machine..../index.txt [hunk 2]
```diff
@@ -1,18 +0,0 @@
----
```
Comment (bridgetkromhout) [inline site/content/events/2011-bangalore/proposals/Agile catalyst : Devops, It starts at Developers machine..../index.txt]: You should not be removing this file.

File: site/content/events/2013-atlanta/proposals/The DevOps Pay Raise: Quantifying Your Value to Move Up the Ladder/index.txt [hunk 3]
```diff
@@ -1,22 +0,0 @@
----
```
Comment (bridgetkromhout) [inline site/content/events/2013-atlanta/proposals/The DevOps Pay Raise: Quantifying Your Value to Move Up the Ladder/index.txt]: Nor this file.

File: site/content/events/2013-atlanta/proposals/Your Agile Performance Testing Checklist: MultiMechanize, Bees with Machine Guns, PageSpeed and AppDynamics/index.txt [hunk 4]
```diff
@@ -1,22 +0,0 @@
----
```
Comment (bridgetkromhout) [inline site/content/events/2013-atlanta/proposals/Your Agile Performance Testing Checklist: MultiMechanize, Bees with Machine Guns, PageSpeed and AppDynamics/index.txt]: Nor this file.

File: site/content/events/2013-austin/proposals/DevStack: Learn OpenStack by running OpenStack/index.txt [hunk 5]
```diff
@@ -1,26 +0,0 @@
----
```
Comment (bridgetkromhout) [inline site/content/events/2013-austin/proposals/DevStack: Learn OpenStack by running OpenStack/index.txt]: Nor this one.

File: site/content/events/2013-downunder/proposals/MYOB: From no ops to devops in 12 easy months/index.txt [hunk 6]
```diff
@@ -1,30 +0,0 @@
----
```
Comment (bridgetkromhout) [inline site/content/events/2013-downunder/proposals/MYOB: From no ops to devops in 12 easy months/index.txt]: Nor this one.

**Comment by bridgetkromhout** [comment]
Hi, @barrychandler - we shouldn't merge this with those file removals and with the merge conflict resolution done the way it is. 

If you look at https://github.com/jedi4ever/devopsdays-webby/pull/2121/files you will see what I mean.

If you want to remove the problematic commits entirely we can at least get the rest merged in; I'm not sure what's going on with your fork. Perhaps you'd be best off making a new fork and then repeating your (minimal) changes in it? Thanks.

**Comment by barrychandler** [comment]
Hi Bridget,

Thanks for getting back to me, I'm sorry I'm new to Git and webby so it's been a bit of  journey for me already.  In addition I was unexpectedly not able to work for a couple of weeks in which time my fork hadn't been updated which probably didn't help.

The other issue was originally I was trying to clone on windows which doesn't work, hence the removal of some other older events content.

Anyway, this morning I merged the latest jedi4ever code with my fork and addressed one conflict which was highlighted.  I'll reapply the files I deleted later and initiate another pull request. I hope that will sort it out.

Let me know if you have any concerns.

Barry

**Comment by barrychandler** [comment]
Hi Bridget 

I took your advise and started again with a new fork although it took me several hours to get my VM working again :(

The new Pull request is #2133 

Thanks 

Barry

**Comment by bridgetkromhout** [comment]
> I took your advise and started again with a new fork although it took me several hours to get my VM working again :(

I'm sorry to hear about that! We did recently have another org add Docker support, which hopefully will help. I'm closing this one without merging, as I've merged the other. Thank you for your efforts.

**Comment by davidasnider** [comment]
Barry, there's a section on using docker in the readme now for using docker.  I haven't tried that particular method, but had written up my own process before that got published...  This process takes all of 5 minutes to get a workable version of a VM with the proper website code to develop against.  You can message me on slack if you have questions on how to do this.

cd <some directory>
git clone <your forked url>
docker run -it --name DevOpsDays -p 8000:80 -v $(pwd)/devopsdays-webby:/vagrant ubuntu:12.04 bash
apt-get update && apt-get -y install sudo vim
bash -x /vagrant/bootstrap.sh
. ~/.profile
cd /vagrant/site
webby rebuild

**Pull request closed without merging**